### PR TITLE
Added docker environment for DEV mode.

### PR DESCRIPTION
**Pull request by tarasinf**

### CODE REVIEW

File: dev/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  links:
+   - db
+db:
+  image: mysql:5.7
+  volumes:
+    - serposcope_db:/var/lib/mysql
+  ports:
+   - "3306:3306"
+  environment:
+    MYSQL_ROOT_PASSWORD: "ROOT_PASS"
```
Comment (Andrii-D) [inline dev/docker-compose.yml]: Don't you need to add $ before each env var?

File: dev/redeploy.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+if [ "$1" = stop ]; then
+    docker-compose stop
```
Comment (Andrii-D) [inline dev/redeploy.sh]: Stop and Rm can be done outside the If statement.

File: dev/docker-compose.yml (hunk 1 above)
Comment (tarasinf) [inline dev/docker-compose.yml]: @Andrii-D  I decided that it's local DB and it's overhead to save credential in env.

File: dev/redeploy.sh (hunk 2 above)
Comment (tarasinf) [inline dev/redeploy.sh]: @Andrii-D Ok

File: dev/docker-compose.yml (hunk 1 above)
Comment (Andrii-D) [inline dev/docker-compose.yml]: So ROOT_PASS - is a local password for database, right?

Comment (tarasinf) [inline dev/docker-compose.yml]: @Andrii-D  "ROOT_PASS" - it's ROOT password for local DB, "DEV_PASS" - it's password for local DB for "DEV_USER".

File: web/src/main/resources/serposcope.conf [hunk 3]
```diff
@@ -9,6 +9,16 @@
...
 # serposcope.db.url=jdbc:mysql://localhost/DATABASE?user=USER&password=PASS&allowMultiQueries=true
+
+# You can get all params for connection to docker DataBase in the next evs:
+# DB_DEV_PORT_3306_TCP_ADDR
+# DB_DEV_PORT_3306_TCP_PORT
+# DB_DEV_ENV_MYSQL_DATABASE
+# DB_DEV_ENV_MYSQL_USER
+# DB_DEV_ENV_MYSQL_PASSWORD
+
+serposcope.db.url=jdbc:mysql://172.17.0.2:3306/DEV_DATABASE?user=DEV_USER&password=DEV_PASS&allowMultiQueries=true
```
Comment (Andrii-D) [inline web/src/main/resources/serposcope.conf]: lowercase

Comment (Andrii-D) [inline web/src/main/resources/serposcope.conf]: Try to use db instead of ip address

File: dev/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    MYSQL_DATABASE: "DEV_DATABASE"
```
Comment (Andrii-D) [inline dev/docker-compose.yml]: use lowercase username and passwords

File: dev/README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+This folder contains the necessary files to develop a Serposcope on any Docker&Docker-Compose - enabled environment, you need to have installed:
+* docker
+* docker-compose
+
+
+
+## COMPILE and RUN
+When we start, we build and run two docker containers. First `dev_serposcope_1` with Serposcope, second `dev_db_1` with DataBase(mysql:5.7). In first run `entrypoint.sh` witch compile and run Serposcope code. Main relations between containers present in `docker-compose.yml`.
+Look configuration of Database in file: `serposcope/web/src/main/resources/serposcope.conf`.
```
Comment (Andrii-D) [inline dev/README.md]: Look for

File: dev/redeploy.sh [hunk 6]
```diff
@@ -0,0 +1,8 @@
+docker-compose stop
+docker-compose rm -f
+
+if [ "$1" = stop ]; then
+    exit 1
+else
```
Comment (Andrii-D) [inline dev/redeploy.sh]: no need

File: web/src/main/resources/serposcope.conf [hunk 7]
```diff
@@ -9,6 +9,16 @@
 
 # alternative database url, mysql example to use mysql : 
 # serposcope.db.url=jdbc:mysql://localhost/DATABASE?user=USER&password=PASS&allowMultiQueries=true
+
+# You can get all params for connection to docker DataBase in the next evns:
```
Comment (Andrii-D) [inline web/src/main/resources/serposcope.conf]: envs

File: dev/README.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
+# Serposcope DEV docker
+
+This folder contains the necessary files to develop a Serposcope on any Docker&Docker-Compose - enabled environment, you need to have installed:
+* docker
+* docker-compose
+
+
+
+## COMPILE and RUN
+When we start, we build and run two docker containers. First `dev_serposcope_1` with Serposcope, second `dev_db_1` with DataBase(mysql:5.7). In first run `entrypoint.sh` witch compile and run Serposcope code. Main relations between containers present in `docker-compose.yml`.
```
Comment (Andrii-D) [inline dev/README.md]: In the first container we are going to compile and run java code each time you do redeploy

**Pull request closed without merging**